src/httpException.filter.ts

```diff
@@ -19,10 +21,11 @@ export class HttpExceptionFilter implements ExceptionFilter {
       | { message: any; statusCode: number; error: number }
       | { error: number; statusCode: 400; message: string[] }; // class-validator 타이핑
 
-    this.logger.debug('@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@');
-    this.logger.debug('status :', status);
-    this.logger.debug('###########################################');
-    this.logger.debug('err :', err);
+    this.mylogger.debug('status : ' + status + ' error : ' + err.message);
```

Comment:
```suggestion
    this.mylogger.error('status : ' + status + ' error : ' + err.message);
```

해당 모듈은 예외를 처리하는 모듈이라 debug보다는 error가 더 좋을 것 같아요. 물론 일반적으로는 error로 레벨을 설정할 거면 메시지를 조금 더 다듬으면 좋긴 하니, 이 부분은 추후에 좀더 고민해봐용